tests/10apidoc/33room-members.pl

```diff
@@ -408,15 +408,38 @@ sub matrix_create_and_join_room
 
 sub room_fixture
 {
+   my ( $user_fixture, %args ) = @_;
+
+   fixture(
+      requires => [ $user_fixture ],
+
+      setup => sub {
+         my ( $user ) = @_;
+
+         matrix_create_room( $user, %args )->then( sub {
+            my ( $room_id ) = @_;
```

Comment:
I was initially unsure why this `then` block exists, but I see it's because of the 2-value return from `matrix_create_room` that also passes the alias back. Could you pop a little comment in here just reminding people of such, so nobody's tempted to remove the block and thus upset most of the tests?